docs/current_docs/quickstart/basics/basics.mdx

```diff
@@ -0,0 +1,288 @@
+
+FIXME
+
+</TabItem>
+<TabItem value="PHP">
+
+FIXME
+
+</TabItem>
+<TabItem value="Java">
+
+FIXME
+
+</TabItem>
+</Tabs>
+
+[FIXME recording]
+
+:::tip FUNCTION NAMES
+When calling Dagger Functions, all names (functions, arguments, fields, etc.) are converted into a shell-friendly "kebab-case" style. This is why a Dagger Function named `FooPublish` in Go, `foo_publish` in Python and `fooPublish` in TypeScript/PHP/Java is called as `foo-publish` in Dagger Shell or on the command-line.
+:::
+
+## Speeding things up with caching
+
+One of Dagger's most powerful features is its built-in caching. This is typically most useful when dealing with package managers such as `apt`, `apk`, `npm`, `maven`, `pip` and similar. For these tools to cache properly, they need their own cache data (usually a directory) to be persisted between runs. Here's a Dagger Function that creates a cache volume to store the packages installed by __:
+
+<Tabs groupId="language">
+<TabItem value="Go">
+
+```go file=./snippets/caching/go/main.go
+```
+
+</TabItem>
+<TabItem value="Python">
+
+FIXME
+
+</TabItem>
+<TabItem value="TypeScript">
+
+FIXME
+
+</TabItem>
+<TabItem value="PHP">
+
+FIXME
+
+</TabItem>
+<TabItem value="Java">
+
+FIXME
+
+</TabItem>
+</Tabs>
+
+Notice that when you call this Dagger Function multiple times, the second and subsequent runs are drastically faster than the first, since Dagger automatically reuses the cached packages instead of re-downloading them every time.
+
+[FIXME recording]
+
+## Using the Daggerverse
+
+You can group Dagger Functions into modules and share them with others - your team, your company, or the broader Dagger community. And just as others can use your modules, you too can use modules created and shared by others, to speed up your development and take advantage of best practices. The Daggerverse is a free service run by Dagger, which indexes all publicly available Dagger modules, and lets you easily search and consume them.
+
+[FIXME screenshot]
+
+Here's an example of installing a module from the Daggerverse and using it to scan a container built by another Dagger Function:
+
+[FIXME recording]
+
+## Using Dagger Cloud
+
+Building and running workflows is only part of the problem - you also need a way to inspect and monitor them. Dagger provides two powerful real-time visualization tools: the Dagger terminal UI (TUI), which you've already seen above, and Dagger Cloud, a browser-based interface focused on tracing and debugging Dagger workflows.
+
+Once configured, every time you execute a Dagger workflow, its operational telemetry is automatically sent to Dagger Cloud as a Trace and the workflow output includes a link to visualize the workflow run on Dagger Cloud. Here's an example:
+
+[FIXME screen recording]
+
+:::note DAGGER CLOUD
+Dagger Cloud sign-up is optional, and free of charge for a single user. A GitHub or Google account is required for account setup and identity verification.
+:::
+
+
+## Next steps
+
+FIXME
```

Comment:
This should be a list of the coding quickstarts

Reply:
Added